tests/unit/lms/services/blackboard_api/client_test.py

```diff
@@ -319,16 +319,18 @@ def test_it(
         )
         assert public_url == blackboard_public_url_schema.parse.return_value
 
-    def test_it_raises_BlackboardFileNotFoundInCourse_if_the_Blackboard_API_404s(
+    def test_it_raises_FileNotFoundInCourse_if_the_Blackboard_API_404s(
         self, svc, basic_client
     ):
         basic_client.request.side_effect = ExternalRequestError(
             response=factories.requests.Response(status_code=404)
         )
 
-        with pytest.raises(BlackboardFileNotFoundInCourse):
+        with pytest.raises(FileNotFoundInCourse) as excinfo:
             svc.public_url("COURSE_ID", "FILE_ID")
 
+        assert excinfo.value.error_code == "blackboard_file_not_found_in_course"
```

Comment:
Checking the error code in all of these to make the change safer.

Reply:
Yeah, I'd say this is a feature worth testing. I'm not a fan of testing human readable strings, but codes are here exactly here for this reason.